Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -625,12 +625,12 @@
     slots:
     - Belt
   - type: Gun
-    fireRate: 1 #Sunrise-Edit
+    fireRate: 2
     soundGunshot:
       path: /Audio/Weapons/Guns/Gunshots/taser.ogg
   - type: ProjectileBatteryAmmoProvider
     proto: BulletTaserSunrise #Sunrise-Edit
-    fireCost: 100 #Sunrise-Edit
+    fireCost: 83.3 #Sunrise-Edit 12 Shots
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)